ui/packages/atlasmap/src/Views/MappingTableView.tsx

```diff
@@ -46,25 +47,54 @@ const styles = StyleSheet.create({
 });
 
 export interface IMappingTableProps {
-  mappings: IAtlasmapMapping[];
+  mappings: Array<IAtlasmapMapping>;
   onSelectMapping: (mapping: IAtlasmapMapping) => void;
+  shouldShowMappingPreview: (field: IAtlasmapMapping) => boolean;
+  onFieldPreviewChange: (field: IAtlasmapField, value: string) => void;
 }
 
 export const MappingTableView: FunctionComponent<IMappingTableProps> = ({
   mappings,
   onSelectMapping,
+  shouldShowMappingPreview,
+  onFieldPreviewChange,
 }) => {
+  const renderPreview = (
+    mapping: IAtlasmapMapping,
+    mappedField: IAtlasmapField,
+  ) =>
+    mappedField &&
```

Comment:
This check isn’t really needed since the type doesn’t allow for null

Reply:
Ah, that was a holdover from the previous code. One minute...